src/help.ts

```diff
@@ -76,4 +91,63 @@ export const FUNCTION_HELP_DEFINITIONS: Record<string, string[]> = {
     'Function that compares two `decimal` operands and evaluates to `true` if the left operand is numerically greater than or equal to the right operand.' +
       '\n\nhttps://docs.cedarpolicy.com/policies/syntax-operators.html#function-greaterThanOrEqual',
   ],
+  // Datetime extension functions
+  datetime: [
+    'datetime(String): datetime',
+    '*(experimental)* Function that constructs a `datetime` value from a string in one of the forms:' +
+      '\n* `"YYYY-MM-DD"` (date only)' +
+      '\n* `"YYYY-MM-DDThh:mm:ssZ"` (UTC)' +
+      '\n* `"YYYY-MM-DDThh:mm:ss.SSSZ`" (UTC with millisecond precision)' +
+      '\n* `"YYYY-MM-DDThh:mm:ss(+/-)hhmm"` (With timezone offset in hours and minutes)' +
+      '\n* `"YYYY-MM-DDThh:mm:ss.SSS(+/-)hhmm"` (With timezone offset in hours and minutes and millisecond precision)' +
+      '\n\nhttps://github.com/cedar-policy/rfcs/blob/main/text/0080-datetime-extension.md',
```

Comment:
It'd be better to link to the Cedar docs:
 - Types can be found in https://docs.cedarpolicy.com/policies/syntax-datatypes.html#datatype-extension
 - Operators can be found in https://docs.cedarpolicy.com/policies/syntax-operators.html#functions-datetime

Reply:
updated help with links